src/pages/AddCluster/KClusterList/index.js

```diff
@@ -125,7 +125,7 @@ export default class EnterpriseClusters extends PureComponent {
                 taskID: task.taskID,
                 status: infos[0].state,
                 message: infos[0].message,
-                install_step: provider,
+                install_step: 'createK8s',
```

Comment:
What's the log phase here?

Reply:
The first phase starts installing the cluster